etl/steps/data/garden/wb/2025-04-14/world_bank_pip_dimensional.meta.yml

```diff
@@ -0,0 +1,928 @@
+    Extreme poverty is defined as living below the International Poverty Line of {definitions.poverty_line}. {definitions.ppp_adjustment_subtitle}
+    <%- elif poverty_line in ["320", "365"] %>
+    The poverty line of {definitions.poverty_line} is set by the World Bank to be representative of the definitions of poverty adopted in lower-middle-income countries. {definitions.ppp_adjustment_subtitle}
+    <%- elif poverty_line in ["550", "685"] %>
+    The poverty line of {definitions.poverty_line} is set by the World Bank to be representative of the definitions of poverty adopted in upper-middle-income countries. {definitions.ppp_adjustment_subtitle}
+    <%- elif poverty_line in ["40% of median", "50% of median", "60% of median"] %>
+    Relative poverty is measured in terms of a poverty line that rises and falls over time with average incomes – in this case set at {definitions.relative_poverty_percentage} of the median <<welfare_type>>.
+    <%- else %>
+    {definitions.ppp_adjustment_subtitle}
+    <%- endif %>
+
+  note_ppp: |-
+    <% if welfare_type == "income or consumption" %>
+    {definitions.ppp_adjustment_note} Depending on the country and year, it relates to income measured after taxes and benefits, or to consumption, [per capita](#dod:per-capita).
+    <%- elif welfare_type == "income" %>
+    {definitions.ppp_adjustment_note} It relates to income measured after taxes and benefits [per capita](#dod:per-capita).
+    <%- elif welfare_type == "consumption" %>
+    {definitions.ppp_adjustment_note} It relates to consumption [per capita](#dod:per-capita).
+    <%- endif %>
+
+  note_without_ppp: |-
+    <% if welfare_type == "income or consumption" %>
+    Depending on the country and year, the data relates to income measured after taxes and benefits, or to consumption, [per capita](#dod:per-capita).
+    <%- elif welfare_type == "income" %>
+    The data relates to income measured after taxes and benefits [per capita](#dod:per-capita).
+    <%- elif welfare_type == "consumption" %>
+    The data relates to consumption [per capita](#dod:per-capita).
+    <%- endif %>
+
+  note_for_absolute_or_relative_poverty: |-
+    <% if poverty_line in ["40% of median", "50% of median", "60% of median"] %>
+    {definitions.note_without_ppp}
+    <%- else %>
+    {definitions.note_ppp}
+    <%- endif %>
+
+  hasmaptab: |-
+    <% if survey_comparability == "No spells" %>
+    true
+    <%- else %>
+    false
+    <%- endif %>
+
+  tab: |-
+    <% if survey_comparability == "No spells" %>
+    map
+    <%- else %>
+    chart
+    <%- endif %>
+
+  headcount_ratio_color_scheme: |-
+    <% if poverty_line in ["40% of median", "50% of median", "60% of median"] %>
+    YlOrBr
+    <%- else %>
+    OrRd
+    <%- endif %>
+
+  map_brackets_headcount_ratio: &map_brackets_headcount_ratio
+    <% if poverty_line in ["190", "215"] %>
+    [3, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100]
+    <%- elif poverty_line in ["40% of median"] %>
```

Comment:
Hi @Marigold! How can I make these jinja-brackets work? I get an error that this is not an array? Would we need to add an exception for this category, as you've done in the past?

Reply:
And I think there is a similar issue with map and hasmap tab, but it doesn't break the code

Reply:
I'll see what I can do.

Reply:
Check this out https://github.com/owid/etl/pull/4369